pyicloud/services/photos.py

```diff
@@ -7,7 +7,10 @@
 from pyicloud.exceptions import PyiCloudServiceNotActivatedException
 from pytz import UTC
 
-from future.moves.urllib.parse import urlencode
+try:
+    from future.moves.urllib.parse import urlencode
```

Comment:
this was import causing an error for me using Python 3.7.4

Reply:
Be my guest?

Here's the trace.
```
$ python
Python 3.7.4 (default, Sep  3 2019, 06:39:49)
[Clang 10.0.0 (clang-1000.11.45.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from future.moves.urllib.parse import urlencode
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'future'
>>> from urllib.parse import urlencode # success
>>>
```

Reply:
I believe you are having an error! Don't be mad!

I'm just saying that it should not be in this PR.

And maybe you are having this because you have not installed the requirements ?

Having Python 3.7.6 but don't use photo service, tests should be added so we can see it in the CI.

Fixed on #268

Reply:
`future` is listed as a dependency in `requirements.txt` and I think when I ran the tests in my python3 execution environment I didn't have that dep. fine w/ removing this diff.